sys/Makefile.dep

```diff
@@ -1,7 +1,3 @@
-ifneq (,$(filter eepreg,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_eeprom
-endif
```

Comment:
Oh, this is causing the compilation issues detected by Murdock. Sorry for overlooking this.

The dependency to `periph_eeprom` needs to remain.